envoy/upstream/upstream.h

```diff
@@ -991,6 +1031,21 @@ class ClusterInfo : public Http::FilterChainFactory {
    */
   virtual TransportSocketMatcher& transportSocketMatcher() const PURE;
 
+  /**
+   * @return ClusterConfigUpdateStats& strongly named config update stats for this cluster.
```

Comment:
I realize this phrase 'strongly named' pre-existed, but I'm not sure what it means.

It's possible I came up with phrase and i still don't know what it means :)

Do you? If not, can we just remove this here & below?

For ClusterLbStats maybe use 'load-balancer-related stats for this cluster'.

Reply:
Done.